docs/fiddles/windows/manage-windows/New Window Fiddle/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+</head>
+<body>
+  <h2>Create a new window</h2>
+  <h3>Supports: Win, MacOS, Linux <span>|</span> Process: Main</h3>
+  <button id="new-window">View Demo</button>
+  <p>The <code>BrowserWindow</code> module gives you the ability to create new windows in your app. This main process module can be used from the renderer process with the <code>remote</code> module, as is shown in this demo.</p>
+  <p>There are a lot of options when creating a new window. A few are in this demo, but visit the <a href="http://electron.atom.io/docs/api/browser-window">documentation<span class="u-visible-to-screen-reader">(opens in new window)</span></a>
```

Comment:
Would need to add an event listener to this anchor tag that calls the `shell.openExternal` API with a snippet like below in order to open the page in a browser window.

```js
link.addEventListener('click', (e) => {
      e.preventDefault()
      shell.openExternal(url)
    })
```

Reply:
Thanks for that!